apps/web/components/NewsArticle/NewsArticle.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react'
```

Comment:
Is this component specific for institutions? if so I'm just wondering if the name is too generic and could lead to some misunderstanding

Reply:
I think the idea was to use the same wrapper for both frontpage news and organization news in the future. But for now I've moved it under `components/organization/NewsArticle`